test/integration-test.sh

```diff
@@ -106,6 +106,43 @@ validate_node_balance_gt0 "${api2}"
 validate_node_balance_gt0 "${api3}"
 echo "- Nodes are funded"
 
+# should fail authentication without proper token
```

Comment:
Can we separate this into a hoprd-test.sh - I don't want to mix up testing the protocol, and testing the tooling.